Sources/TuistGraphTesting/Models/TextSettings+TestData.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+@testable import TuistGraph
+
+public extension TextSettings {
+    static func test(indentWidth: UInt? = 2, tabWidth: UInt? = 2) -> TextSettings {
+        TextSettings(usesTabs: true, indentWidth: indentWidth, tabWidth: tabWidth, wrapsLines: true)
```

Comment:
nit: I'd add `usesTabs` and `wrapsLines` to the parameters of this method as well